<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remove_employee</class>
 <widget class="QDialog" name="remove_employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Employee</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>492</width>
     <height>303</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="empId"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Employee ID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="emp_fn"/>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QTableView" name="emp_view"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="emp_ln"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="view_emp">
      <property name="text">
       <string>View Employees</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="rem_emp">
      <property name="text">
       <string>Remove Employee</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>remove_employee</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
